models/servers.js

```diff
@@ -1,6 +1,7 @@
 var Backbone = require('backbone');
 
 var ServersCollection = Backbone.Collection.extend({
+/*<<<<<<< HEAD:models/servers.js
```

Comment:
This commented out section will stop the buttons from starting, stopping, and getting players.